site/src/patterns/chat.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Chat
+description: The chat pattern in the Atlas Design System
+template: standard
+---
+
+# Chat Pattern
+
+The message conversation would be used in the following manner.
+
+```html
+<section class="display-grid gap-sm">
+	<article class="display-grid">
+		<div class="message">
+			<p class="message-time">May 29, 2023, 4:50PM</p>
+			<div class="persona message-persona">
+				<figure class="persona-avatar">
+					<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+				</figure>
+			</div>
+			<div class="message-content">
+				<div class="persona message-content-header">
```

Comment:
Please separate `persona` and `message-content-header` as it seems to also contain the popover. Apply the same pattern to all examples including the components/message

Reply:
updated